config/initializers/00_extensions.rb

```diff
@@ -46,6 +46,3 @@ class Integer
   include Extensions::IntegerExtension
 end
 
-class NilClass
-  include Extensions::NilExtension
-end
```

Comment:
🎉 🎉 🎉

Reply:
I've only just seen your edit to this 🤦  I've deleted the extension now too.